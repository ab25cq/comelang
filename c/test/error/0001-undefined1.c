/*
PATTERN: undefined
PATTERN: c:9:5
*/

int
main()
{
	x;
}
