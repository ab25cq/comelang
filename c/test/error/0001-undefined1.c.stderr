undefined symbol x at test/error/0001-undefined1.c:9:5
    x;
    ^
